docs/contributing/implementation-notes.md

```diff
@@ -11,6 +11,8 @@ This section is a collection of implementation notes for the [stack reconciler](
 
 It is very technical and assumes a strong understanding of React public API as well as how it's divided into core, renderers, and the reconciler. If you're not very familiar with the React codebase, read [the codebase overview](/react/contributing/codebase-overview.html) first.
 
+It also assumes a grasp on the [difference between React components, their instances, and elements](/react/blog/2015/12/18/react-components-elements-and-instances.html).
```

Comment:
Can you reword it to:
> It also assumes an understanding of the differences between React components, their instances, and elements

So it's more stylistically consistent with the rest of the content? Thanks!